src/components/store/types/ActionType.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Available action types
+ */
+export enum ActionType {
+  CREATE_BLOCK = 'CREATE_BLOCK',
```

Comment:
```suggestion
  CreateBlock = 'CREATE_BLOCK',
```

See [Enums](https://www.typescriptlang.org/docs/handbook/enums.html)

Reply:
do we need string representations?

Reply:
and also, add jsdocs please for every case

Reply:
I use BlockMutationType instead